src/components/HelpDropdown.tsx

```diff
@@ -0,0 +1,57 @@
+import { ArrowRightOutlined } from "@ant-design/icons";
+import { Button, ConfigProvider, Dropdown, MenuProps, Space } from "antd";
+import React, { ReactElement, useRef } from "react";
+import styled from "styled-components";
+
+import { DropdownSVG } from "../assets";
+import { FlexRowAlignCenter, VisuallyHidden } from "../styles/utils";
+
+const HelpButton = styled(Button)``;
+
+export default function HelpDropdown(): ReactElement {
+  const dropdownContainer = useRef<HTMLDivElement>(null);
+
+  const items: MenuProps["items"] = [
+    {
+      key: "https://github.com/allen-cell-animated/nucmorph-colorizer/issues",
+      label: (
+        <Space>
+          <p>
+            Report an issue
+            <VisuallyHidden>(opens in new tab)</VisuallyHidden>
+          </p>
+          <ArrowRightOutlined />
+        </Space>
+      ),
+    },
+  ];
+  const menuConfig: MenuProps = {
+    items,
+    // Makes dropdown accessible via keyboard nav + enter key
+    onClick: (e) => {
+      window.open(e.key, "_blank", "noopener noreferrer");
+    },
+  };
+
+  return (
+    <ConfigProvider theme={{ components: { Button: { paddingInline: 12 } } }}>
+      <div ref={dropdownContainer}>
+        <Dropdown
+          menu={menuConfig}
+          getPopupContainer={dropdownContainer.current ? () => dropdownContainer.current! : undefined}
+          trigger={["click", "hover"]}
```

Comment:
If you quickly move the cursor into place and click, the hover event can occur just before the click which opens and then immediately closes the pop up. Once I knew the hover would open the pop-up it became intuitive, but that wasn't my assumption at first, so there was some friction for me.

Reply:
Hmm... I solved this problem on the other Dropdowns, but not on this one. I'm considering making a flexible base Dropdown component that fixes this, and I might break that out into a separate PR so it doesn't clutter that one. Will report back once that's done!

Reply:
I had the same experience and was looking for a place to comment on it. I'm sure there's an awkward interaction with the accessibility work you've done though, so I'm comfy with merging this as-is if that tension can't be resolved.